index.html

```diff
@@ -1,23 +1,49 @@
-<!DOCTYPE html>
-<html>
+<!doctype html>
+ <html lang="en">
+ <head>
+ <meta charset="utf-8">
+ <title>income</title>
+ <link rel="stylesheet" href="//code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css">
+ <script src="//code.jquery.com/jquery-1.10.2.js"></script>
+ <script src="//code.jquery.com/ui/1.11.4/jquery-ui.js"></script>
+ <script type="javascript" src= "C:/Users/clara/Desktop/jquery-1.11.0.min.js"></script>
+ <link rel="stylesheet" href="/resources/demos/style.css">
```

Comment:
I couldn't find the style.css in this project.
Please commit patches with necessary files.